bin/server.js

```diff
@@ -16,39 +16,46 @@ const args = require('yargs')
 const express = require('express');
 const http = require('http');
 const Path = require('path');
+const Logger = require('../lib/logger');
+const app = express();
 
 // Load nconf into the global namespace
 global.Config = require('nconf')
     .env()
     .argv();
+
 if (args.c) {
   global.Config.file(Path.resolve(process.cwd(), args.c));
 }
 global.Config.defaults(require('../config/defaults.json'));
 
-global.Log = require('../lib/logger').attach(global.Config);
-const PluginManager = require('../lib/plugin-manager');
+// Set up logging
+global.Log = Logger.attach(global.Config.get('log:level'), global.Config.get('log:filename'));
 
-const host = Config.get('service:hostname');
-const port = Config.get('service:port');
-const app = express();
-const server = http.createServer(app);
-const storage = new (require('../lib/storage'))();
-const manager = new PluginManager(storage);
+// Add request logging middleware
+const accessLog = Logger.attach(global.Config.get('log:access:level'), global.Config.get('log:access:filename'));
 
-manager.on('error', (err) => {
-  Log.error(err);
-});
+app.use(Logger.logRequests((message) => accessLog.verbose(message)));
```

Comment:
What's the `.verbose` method doing? Does that mean messages are only logged at the verbose level?